WhisperAPI/WhisperAPI/Models/ConversationContext.cs

```diff
@@ -10,15 +11,30 @@ public ConversationContext(Guid chatkey, DateTime datetime)
         {
             this.ChatKey = chatkey;
             this.StartDate = datetime;
+            this.MessagesSuggestions = new List<MessageSuggestion>();
         }
 
         private ConversationContext()
         {
+            this.MessagesSuggestions = new List<MessageSuggestion>();
         }
 
         [Key]
         public Guid ChatKey { get; set; }
 
         public DateTime StartDate { get; set; }
+
+        public ICollection<MessageSuggestion> MessagesSuggestions { get; set; }
+
+        public string GetAllMessages()
+        {
+            var allMessages = string.Empty;
+            if (this.MessagesSuggestions != null)
```

Comment:
Perhaps we don't need null-check. But if you do null checks, prefer the fail-fast approach. It looks like this:

public string GetAllMessages()
{
    if(MessagesSuggestions == null)
        return "";
    //return String.Join(...)
}

The philosophy is that you take care early in the method of everything that could go wrong (you throw exceptions, etc.) and after that, the rest of the method is cleaner, because you know that what you have is right and its indentation level start at 0 (you are not in a if or something). 
There is a conversation on this here:
https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement